src/test/resources/testResult/featureCollection/monitoring-locations/featureCollection_bbox_one.json

```diff
@@ -21,6 +21,12 @@
           "rel": "collection",
           "title": "NWIS Monitoring Locations",
           "type": "application/json"
+        },
+        {
+          "rel": "collection",
+          "href": "https://unittest/api/observations/collections/ANC?f=json",
```

Comment:
Should this one be the same?

Reply:
Oh, I notice this change is in a PR by @orosz-usgs that is currently open as well.  Ed you probably know this better than I do, it looks like we won't collide since we added the same change.

Reply:
yes, it is the same change I have pending in another pull request.  I'll merge this first and resolve any conflicts.